### ISSUE TITLE
Chat2: Issue with auto-scroll when using modern layout

### ISSUE THREAD

**Issue by Fractura** [ascclemens/plugin-issues#14]
There is an issue that causes the auto-scroll to freak out if the topmost visible line is an empty line.

Affected settings: "Use modern layout" (on), "Hide timestamps when redundant" (off)

Steps to reproduce:
1. Enter a command that produces empty lines as part of it, for example `/? hotbar`
2. Resize the window to the point where the topmost visible chat line would be an empty line OR write lines into chat until the empty line is reaching the top.
3. Expected behavior: Nothing happens, actual behavior: the chat repeatedly "skips" up a line, scrolls back down, repeat.
See: https://imgur.com/a/QfkG2GB

Notes: 
- When not using modern layout, empty lines like this cause the timestamp to appear twice (in the same line)
- This issue does **not** occur when "Hide timestamps when redundant" is active, as the line is then removed in its entirety - this is a valid workaround for now though I'm not sure dropping empty lines is the intended behavior as they are placed for readability.

**Comment by ascclemens** [comment]
Put another line in chat, like `/e 1` or something. This issue has been around forever and is very difficult to pinpoint.

**Comment by Fractura** [comment]
I figured I could "stop" the effect by producing another chat line by any means necessary. But digging a bit deeper today I figured out I can use the timestamp-hiding to stop this from occurring as well (at the cost of losing the empty lines altogether, but oh well).

If it's known and hard to pinpoint/"won't fix", feel free to close it; or leave it open so people can see it when they come here, whatever suits you best.

Thanks and have a nice day!
